tests/tests_e3/anod/test_qualifier_manager.py

```diff
@@ -171,13 +172,20 @@ class AnodDummy(Anod):
     with pytest.raises(AnodError) as err:
```

Comment:
Now it should raise a QualifierError right ?

Reply:
I have updated this every where I found it